learning_assistant/management/commands/set_course_prompts.py

```diff
@@ -0,0 +1,108 @@
+"""
+Django management command to generate course prompts.
+"""
+import json
+import logging
+from posixpath import join as urljoin
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from edx_rest_api_client.client import OAuthAPIClient
+from opaque_keys.edx.keys import CourseKey
+
+from learning_assistant.models import CoursePrompt
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django Management command to create a set of course prompts
+    """
+
+    def add_arguments(self, parser):
+
+        # list of course ids
+        parser.add_argument(
+            '--course_ids',
+            dest='course_ids',
+            help='Comma separated list of course_ids to generate. Only newer style course ids can be supplied.',
+        )
+
+        # pre-message
+        parser.add_argument(
+            '--pre_message',
+            dest='pre_message',
+            help='Message to prepend to course topics',
+        )
+
+        parser.add_argument(
+            '--skills_descriptor',
+            dest='skills_descriptor',
+            help='Message that describes skill structure'
+        )
+
+        # post-message
+        parser.add_argument(
+            '--post_message',
+            dest='post_message',
+            help='Message to append to course topics',
+        )
+
+    @staticmethod
+    def _get_discovery_api_client():
+        """
+        Returns an API client which can be used to make Catalog API requests.
+        """
+        return OAuthAPIClient(
+            base_url=settings.DISCOVERY_BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL,
+            client_id=settings.DISCOVERY_BACKEND_SERVICE_EDX_OAUTH2_KEY,
+            client_secret=settings.DISCOVERY_BACKEND_SERVICE_EDX_OAUTH2_SECRET,
+        )
+
+    def handle(self, *args, **options):
+        """
+        Management command entry point.
+
+        This command is meant to generate a small (<500) set of course prompts. If a larger number of prompts
```

Comment:
Maybe I'm missing this, but does the function actually stop at 500 prompts, or is this meant to be a suggestion?

Reply:
This is more of a suggestion!